frontend/src/framework/internal/components/LoginButton/loginButton.tsx

```diff
@@ -29,6 +29,16 @@ export const LoginButton: React.FC<LoginButtonProps> = (props) => {
 
     function makeIcon() {
         if (authState === AuthState.LoggedIn) {
+            if (userInfo?.avatar_b64str) {
+                return (
+                    <img
+                        src={`data:image/png;base64,${userInfo.avatar_b64str}`}
+                        alt="Avatar"
+                        className="w-4 h-4 rounded-full"
```

Comment:
Should we make the picture slightly larger, same size as icon? 
```suggestion
                        className="w-5 h-5 rounded-full"
```